manifests/application/migrator.pp

```diff
@@ -22,31 +26,31 @@
 # $primary_client_metadata_uri       - see lightblue::client::configure
 # $primary_client_data_uri           - see lightblue::client::configure
 # $primary_client_use_cert_auth      - see lightblue::client::configure
-# $primary_client_ca_file_path       - see lightblue::client::configure
+# $primary_client_ca_file_path       - (optional) ca file path
 # $primary_client_ca_source          - (optional) source for ca file.
-# $primary_client_cert_file_path     - see lightblue::client::configure
+# $primary_client_cert_file_path     - (optional) cert file path
 # $primary_client_cert_source        - (optional) source for cert file.
 # $primary_client_cert_password      - see lightblue::client::configure
 # $primary_client_cert_alias         - see lightblue::client::configure
 #
-# $source_config                     - Source lightblue client configuration file. Defaults to $config_file.
+# $is_source_same_as_primary         - Indicates if the source lightblue instance is the same as the primary. Defaults to true.
```

Comment:
Two things:
1. this has nothing to do with the context of the PR
2. why not just check if source config = primary config, like it was doing before?